File: dev-docs/snippets/databases/update.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Update a database's title or type.
export async function update(client: JsBaoClient, databaseId: string) {
  // #region example
  const db = await client.databases.update(databaseId, {
    title: "Renamed catalog",
  });
  // #endregion example
  return db;
}

File: dev-docs/snippets/databases/update.swift
import JsBaoClient

// Update a database's title or type.
func update(client: JsBaoClient, databaseId: String) async throws {
  // #region example
  let db = try await client.databases.update(
    databaseId: databaseId,
    params: UpdateDatabaseParams(title: "Renamed catalog")
  )
  // #endregion example
  _ = db
}
